frontend/src/metabase/admin/settings/components/widgets/SettingSelect.jsx

```diff
@@ -15,6 +15,9 @@ const SettingSelect = ({
     defaultValue={defaultValue}
     searchProp={searchProp}
     onChange={e => onChange(e.target.value)}
+    buttonProps={{
+      dataTestId: key,
+    }}
```

Comment:
This was done to help make testing easier. I did need to adjust 1 cypress test because of this change